lib/DBDish.pm6

```diff
@@ -19,6 +19,17 @@ role Driver does DBDish::ErrorHandling {
     #method new() { nextsame; }
 }
 
+role Type {
+    has Sub %!Conversions{Str};
```

Comment:
This probably wants to be relaxed to Routine or even Callable, using Sub here is a bit specific I think :)